.github/workflows/TICKET-MATE-BE-SCHEMA-VERIFY.yaml

```
@@ -0,0 +1,201 @@
+              pg_isready -h localhost -p 5432 -U postgres; then
+              echo "✅ Postgres 준비 완료"
+              exit 0
+            fi
+            echo "⏳ DB 준비 대기중... ($i/30)"
+            sleep 2
+          done
+          echo "❌ Postgres가 준비되지 않았습니다." && exit 1
+
+      - name: 마이그레이션 파일 개수 확인 (디렉토리 유무 포함)
+        id: migcheck
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:7:28: Double quote to prevent globbing and word splitting

(shellcheck)